geometry_msgs/PoseWithCovarianceStamped[] poseMsgs
string carId